src/views/login/login.view.tsx

```diff
@@ -40,11 +44,23 @@ const Login: FC = () => {
     }
   }, [account, state, navigate]);
 
+  if (!env) {
+    return null;
+  }
+
+  const deploymentName = getDeploymentName(env.chains[0]);
+  const appName = deploymentName ? `Bridge ${deploymentName}` : "Bridge";
+  const networkName = getNetworkName(env.chains[0]);
+  const networkInfo =
+    networkName !== undefined &&
+    networkName !== "Mainnet" &&
```

Comment:
Why not checking again the `chainId` instead of checking again a label? Then we can remove the `as const` from the `getNetworkName` function and just return a string (which makes more sense for a label as well).

Reply:
Well, the function `getNetworkName` exists to abstract its user from knowing about chainId's. If we start comparing chainId's we can remove the function. I think computing the network's name and then deciding which names I want to react to is a better approach. Why don't you like using `as const`? I think it's perfectly ok that a label is inferred as a subset of string literals.

Reply:
But in this case the user won't read the `getNetworkName` function... And for us is better and easier to compare chain ids than calculating a label of the UI (which is calculated based on the chain id) and then comparing against this label...

Reply:
As agreed, I created the `EthereumChainId` enum and used chainId to check for the network.